Pod/Classes/ABEIssue.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ABEIssue.swift
+//  Pods
+//
+//  Created by Hakon Hanesand on 10/6/16.
+//
+//
+
+struct ABEIssue {
+   
+    var title: String?
+    var issueDescription: String?
+    
+    var imageURLS : [String] = []
+    
+    mutating func attachImage(withURL urlString: String) {
+        imageURLS.append(urlString)
+    }
+    
+    var textRepresentation : String? {
+        // TODO
```

Comment:
I was a todo for me to finish up `extraDebuggingInformation`, and I did it on the error_handling branch 👍